pkg/testcase/selinuxcontext_model.go

```diff
@@ -0,0 +1,456 @@
+package testcase
```

Comment:
this file should now live elsewhere since its not a testCase....lets add it someqhere else, like assert pkg or factory , which one you think its best ? 
or even creating a new dir inside pkg ( model )  idk . 
but here makes no sense

Reply:
when deciding that remember the pkgs purposes:

- assert - place to create custom functions to enhance our power of assertions throughout all the framework 
- factory - place to build stuff that needs to be instanciated to be reusable during the run time
- model - place to have the logic data modeling or somwething similar